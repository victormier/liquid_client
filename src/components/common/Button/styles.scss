.button {
  padding: 25px 40px;
  background: white;
  color: black;
  border: none;
  width: 100%;
  border-radius: 60px;
}
