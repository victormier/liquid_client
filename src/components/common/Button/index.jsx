import React from 'react';
import PropTypes from 'prop-types';
import styles from './styles.scss';

const Button = props => (
  <button className={styles.button}>{ props.text }</button>
);

Button.propTypes = {
  text: PropTypes.string.isRequired,
};


export default Button;
