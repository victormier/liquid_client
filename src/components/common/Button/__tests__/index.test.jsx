import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Button from '../index';


describe('component', () => {
  it('renders', () => {
    const container = shallow(
      <Button text="test me" />
    );

    expect(shallowToJson(container)).toMatchSnapshot();
  });
});
