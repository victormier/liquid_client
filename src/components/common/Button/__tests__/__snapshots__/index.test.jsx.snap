// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders 1`] = `
<button>
  test me
</button>
`;
